pkg/build/generator/generator_test.go

```diff
@@ -370,6 +370,38 @@ func TestInstantiateWithImageTrigger(t *testing.T) {
 				bc = buildConfig
 				return nil
 			}
+		client.GetImageStreamFunc =
+			func(ctx kapi.Context, name string) (*imageapi.ImageStream, error) {
+				return &imageapi.ImageStream{
+					ObjectMeta: kapi.ObjectMeta{Name: name},
+					Status: imageapi.ImageStreamStatus{
+						DockerImageRepository: originalImage,
+						Tags: map[string]imageapi.TagEventList{
+							"tag1": {
+								Items: []imageapi.TagEvent{
+									{
+										DockerImageReference: "ref@" + name + ":tag1",
```

Comment:
@bparees @miminar i think it was like this before, I can fix it to reflect the reality